CenterForOpenScience/osf.io
# [OSF-6904]Add Location and Date to Meetings pages

## Purpose
Display the datetime and location of each meeting on meeting landing page. And fix the populate_meeting.py since the required field is_meeting is not added for each meeting.



## QA notes:
1. goes to osf.io/meetings
2. you can see some meetings have location and datetime information 
![screen shot 2016-11-03 at 2 23 49 pm](https://cloud.githubusercontent.com/assets/4974056/19979530/2e37e95a-a1d1-11e6-9fb6-f3e0219c38a2.png)
Before the change:
click on one of the meeting to go to the meeting landing page
you will see
![screen shot 2016-11-03 at 2 24 27 pm](https://cloud.githubusercontent.com/assets/4974056/19979571/51136b98-a1d1-11e6-82d9-c8e3ace9d055.png)
It doesn't have any information about meetings' location and datetime
After the change:
The meeting's location and datetime will be showing up in meetings landing page
![screen shot 2016-11-03 at 2 26 02 pm](https://cloud.githubusercontent.com/assets/4974056/19979617/79ffda82-a1d1-11e6-844b-a28414344489.png)
![screen shot 2016-11-03 at 2 26 17 pm](https://cloud.githubusercontent.com/assets/4974056/19979624/827064fc-a1d1-11e6-8a91-95f9466c30b4.png)





## Side effects




## Ticket


https://openscience.atlassian.net/browse/OSF-6904
